src/Microsoft.DotNet.XHarness.iOS.TestRunner/Microsoft.DotNet.XHarness.iOS.TestRunner.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Folder Include="Core\" />
+    <Folder Include="xUnit\" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Remove="xUnit\NUnit3Xml.xslt" />
+    <None Remove="xUnit\NUnitXml.xslt" />
+  </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="xUnit\NUnit3Xml.xslt">
+      <XlfSourceFormat></XlfSourceFormat>
+      <XlfOutputItem></XlfOutputItem>
+    </EmbeddedResource>
+    <EmbeddedResource Include="xUnit\NUnitXml.xslt">
+      <XlfSourceFormat></XlfSourceFormat>
+      <XlfOutputItem></XlfOutputItem>
+    </EmbeddedResource>
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="xunit.analyzers" Version="0.10.0" />

```

Comment:
This might be problematic, it should be synced over DARC to align with dotnet/arcade repo which owns xunit

Reply:
Hit me up, Manuel, about this, I can give you more information on how this works